examples/networking/xdp/xdp_drop_count.py

```diff
@@ -0,0 +1,123 @@
+from bcc import BPF
+import time
+import sys
+
+
+
+
+
+if (len(sys.argv) != 2):
+    print("Usage: {0} <ifdev>\n\ne.g.: {0} eth0".format(sys.argv[0]))
+    exit(1)
+
+
+device = sys.argv[1]
+
+
+# load BPF program
+b = BPF(text = """
+    #include <uapi/linux/bpf.h>
+    #include <linux/in.h>
```

Comment:
I also get a warning and I could not figure out the reason for it, can you give some insights why this silences the warning?